dom/nodes/Document-Element-getElementsByTagName.js

```diff
@@ -190,4 +190,19 @@ function test_getElementsByTagName(context, element) {
     get_elements(context);
     assert_array_equals(actual, expected);
   }, "getElementsByTagName('*')")
+
+  test(function() {
+    var t1 = element.appendChild(document.createElement("abc"));
+    this.add_cleanup(function() {element.removeChild(t1)});
+
+    var l = context.getElementsByTagName("abc");
+    assert_true(l instanceof HTMLCollection);
+    assert_equals(l.length, 1);
+
+    var t2 = element.appendChild(document.createElement("abc"));
+    assert_equals(l.length, 2);
+
+    element.removeChild(t2);
+    assert_equals(l.length, 1);
+  }, "getElementsByTagName() return an HTMLCollection is liveness");
```

Comment:
The test description looks weird grammatically.
Should it be "getElementsByTagName() should be a live collection" or something?

This comment should be applied to all of new tests.

Reply:
Thanks, @tkent-google, updated the test descriptions.